src/Compilers/VisualBasic/Portable/Symbols/Source/SourceNamedTypeSymbol.vb

```diff
@@ -545,6 +542,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
                         Else
                             ' Neither of types is embedded.
+                            Dim _6thArg As Object = Me.ContainingSymbol.ToErrorMessageArgument(ERRID.ERR_TypeConflict6)
```

Comment:
I think we should inline this local, there is no good reason to create the argument before we decided that we need it.